public/sw_user_cache.js

```diff
@@ -93,6 +93,8 @@ async function createRemoteUserLinks() {
             `${apiSource}/schedule/${profile.AD_Username}/`,
             `${apiSource}/myschedule/${profile.AD_Username}/`,
             `${apiSource}/schedulecontrol/${profile.AD_Username}/`,
+            `${apiSource}/schedulecontrol/${profile.AD_Username}/`,
+            `${apiSource}/studentemployment/`,
           );
```

Comment:
Adds the Experience Transcript data is added to the cache.
Btw, I do see that line 95 and 96 are duplicates and that has been fixed :)